db/migrate/20180129231309_add_board_members.rb

```diff
@@ -0,0 +1,131 @@
+class AddBoardMembers < ActiveRecord::Migration[5.0]
+  def up
+    TeamMember.create!(
```

Comment:
Awesome job on assembling all of this.  A few thoughts:

1. Can we shift this over to a `rake` task, instead of a migration?  Preferring to make database alterations in a migration file, and data changes in a `rake` task.
2. In the `rake` task, can you check for the presence of the db record, before creating one?  And if you find one, update it.
3. In a separate `rake` task, can you update all of the other `TeamMember`s in the `/team` page with `group: "team"`

Reply:
Yes, there is already a similar rake task that I didn't notice before. I'll put it there.

Reply:
Or, rereading your comment, in a separate rake task.

Reply:
@hpjaj For now I'm going to move the board rake task into the seeds.rb script and remove the rake tasks for seeding boards and teams. It seems redundant to have both and I like having things in a yaml file.

When I finish up with this PR I think I'll do another branch and put that script into separate rake tasks with a :seed task for reseeding everything. That way a dev can run the whole thing or each one individually and we can keep all the data in .yamls.